### PR TITLE
[9.0][ADD] mrp_bom_structure_xlsx_level_1

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
MRP BOM Structure Report XLSX Level 1
=====================================

This module was written to extend the MRP capabilities of Odoo, and
allows users to export the BOM structure to MS Excel .XLSX format, but up to
just the components immediately assigned to the BOM, not exploding the BOM
further.

Usage
=====

* Go to 'Manufacturing / Products / Bill of Materials', select a BOM and go
to 'More / Export BOM Structure to Excel '

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@mreficent @gmeficent can you review?

File: mrp_bom_structure_xlsx_level_1/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+=====================================
+MRP BOM Structure Report XLSX Level 1
+=====================================
+
+This module was written to extend the MRP capabilities of Odoo, and
```
Comment (elicoidal) [inline mrp_bom_structure_xlsx_level_1/README.rst]: This module extends the...

File: mrp_bom_structure_xlsx_level_1/README.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+allows users to export the BOM structure to MS Excel .XLSX format, but up to
+just the components immediately assigned to the BOM, not exploding the BOM
+further.
+
+
+Usage
+=====
+
+* Go to 'Manufacturing / Products / Bill of Materials', select a BOM and go
```
Comment (elicoidal) [inline mrp_bom_structure_xlsx_level_1/README.rst]: better enumerated  list

File: mrp_bom_structure_xlsx_level_1/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    'version': '9.0.1.0.0',
+    'category': 'Manufacturing',
+    'author': "Eficent, Odoo Community Association (OCA)",
+    'website': 'https://github.com/OCA/manufacture-reporting',
+    'license': 'AGPL-3',
+    "depends": ['mrp_bom_structure_xlsx'],
+    "data": [
+        'report/bom_structure_xlsx.xml',
+    ],
+    'test': [],
```
Comment (elicoidal) [inline mrp_bom_structure_xlsx_level_1/__openerp__.py]: remove empty keys

File: mrp_bom_structure_xlsx_level_1/report/bom_structure_xlsx.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import xlwt
+from openerp.addons.mrp_bom_structure_xls.report.bom_structure_xls import \
```
Comment (elicoidal) [inline mrp_bom_structure_xlsx_level_1/report/bom_structure_xlsx.py]: Do you really need to input the full path?

File: mrp_bom_structure_xlsx_level_1/report/bom_structure_xlsx.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import xlwt
```
Comment (elicoidal) [inline mrp_bom_structure_xlsx_level_1/report/bom_structure_xlsx.py]: Do you need to protect the path for error?

File: mrp_bom_structure_xlsx_level_1/static/description/index.html [hunk 6]
```diff
@@ -0,0 +1,82 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Export BOM Structure to Excel</h2>
+            <p>
+                This module was written to extend the MRP capabilities of Odoo,
```
Comment (elicoidal) [inline mrp_bom_structure_xlsx_level_1/static/description/index.html]: This module extends

File: mrp_bom_structure_xlsx_level_1/README.rst [hunk 7]
```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+=====================================
+MRP BOM Structure Report XLSX Level 1
+=====================================
+
+This module was written to extend the MRP capabilities of Odoo, and
+allows users to export the BOM structure to MS Excel .XLSX format, but up to
```
Comment (elicoidal) [inline mrp_bom_structure_xlsx_level_1/README.rst]: allows users to export up to level 1 the BOM structure in MS Excel .XLSX format.

(I would not explain more)

**Review by elicoidal — CHANGES_REQUESTED** [review]
Travis and Runbot to be fixed.

**Comment by JordiBForgeFlow** [comment]
@mreficent Can you please attend to the comments from @elicoidal ?

**Comment by MiquelRForgeFlow** [comment]
Yes, I think that now all his requested changes are attended.

File: mrp_bom_structure_xlsx_level_1/README.rst [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+
+Usage
+=====
+
+To use this module, you need to:
+
+1 - Go to 'Manufacturing / Products / Bill of Materials'
+2 - Select a BOM or more BOMS
+3 - Go to 'More / Export BOM Structure to Excel'.
```
Comment (LoisRForgeFlow) [inline mrp_bom_structure_xlsx_level_1/README.rst]: not showing properly.

**Review by LoisRForgeFlow — APPROVED** [review]
Tested on runbot. :+1:

File: mrp_bom_structure_xlsx_level_1/README.rst (hunk 8 above)
Comment (AaronHForgeFlow) [inline mrp_bom_structure_xlsx_level_1/README.rst]: Same comment as @lreficent you should use #.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM despite the comment above

**Review by gmeficent — APPROVED** [review]
xlsx BOM reports correctly generated on Runbot :+1:

**Pull request merged**